### PR TITLE
Update Helm and add it to CI

### PR DESCRIPTION
**Pull request by nickolaev**
## Description
Update the helm charts to the latest develomeents and add the ORG parameter.
Add helm.mk with make helm-install... and make helm-delete... targets.
Use these new targets in the CI with the examples.

## Motivation and Context
We need CI for the Helm charts and they needed update.

## How Has This Been Tested?

- [x] Covered by existing integration testing
- [ ] Added integration testing to cover
- [ ] Tested locally
- [x] Have not tested

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:

- [ ] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.

### CODE REVIEW

File: .cloudtest/aws.yaml [hunk 1]
```diff
@@ -10,7 +10,7 @@ providers:
     env:
       - CLUSTER_RULES_PREFIX=aws
       - AWS_CLUSTER_NAME=$(cluster-name)-$(date)-${CIRCLE_BUILD_NUM}-$(rands10)
-      - CONTAINER_REPO=networkservicemesh
+      - CONTAINER_REPO=networkservicemeshci
```
Comment (haiodo) [inline .cloudtest/aws.yaml]: I think better to move CONTAINER_REPO into env-check, to just ensure it is passed.

File: .cloudtest/azure.yaml [hunk 2]
```diff
@@ -10,7 +10,7 @@ providers:
     env:
       - CLUSTER_RULES_PREFIX=azure
       - AZURE_CLUSTER_NAME=$(cluster-name)-$(date)-${CIRCLE_BUILD_NUM}-$(rands10)
-      - CONTAINER_REPO=networkservicemesh
+      - CONTAINER_REPO=networkservicemeshci
```
Comment (haiodo) [inline .cloudtest/azure.yaml]: Same here.

File: .cloudtest/gke.yaml [hunk 3]
```diff
@@ -11,7 +11,7 @@ providers:
       - GKE_PROJECT_ID=ci-management
       - CLUSTER_RULES_PREFIX=gke
       - GKE_CLUSTER_NAME=$(cluster-name)-$(date)-${CIRCLE_BUILD_NUM}-$(rands10)
-      - CONTAINER_REPO=networkservicemesh
+      - CONTAINER_REPO=networkservicemeshci
```
Comment (haiodo) [inline .cloudtest/gke.yaml]: And here.

File: .cloudtest/vagrant.yaml [hunk 4]
```diff
@@ -10,7 +10,7 @@ providers:
     env:
       - CLUSTER_RULES_PREFIX=vagrant
       - CONTAINER_BUILD_PREFIX=docker
-      - CONTAINER_REPO=networkservicemesh
+      - CONTAINER_REPO=networkservicemeshci
```
Comment (haiodo) [inline .cloudtest/vagrant.yaml]: and here.

File: .cloudtest/aws.yaml (hunk 1 above)
Comment (nickolaev) [reply .cloudtest/aws.yaml]: OK will try and see how this works.

**Comment by tiswanso** [comment]
The default tag of `master` while the build system defaults to `latest` screwed me up pretty good.

**Pull request merged**